darts/models/forecasting/torch_forecasting_model.py

```diff
@@ -493,7 +506,7 @@ def fit_from_dataset(self,
         train_loader = DataLoader(train_dataset,
                                   batch_size=self.batch_size,
                                   shuffle=True,
-                                  num_workers=0,
+                                  num_workers=num_loader_workers,
```

Comment:
Do you also get warnings for every epoch?
Warning:
[W ParallelNative.cpp:214] Warning: Cannot set number of intraop threads after parallel work has started or after set_num_threads call when using native parallel backend (function set_num_threads)

![image](https://user-images.githubusercontent.com/84317528/141334040-f411d585-e417-4f66-bf69-e807975290d5.png)

to reproduce (also checked with other test cases):
```
from darts.utils.timeseries_generation import linear_timeseries
from darts.models import TFTModel

x = linear_timeseries(length=100, freq='h')
model = TFTModel(input_chunk_length=10, output_chunk_length=10, add_cyclic_encoder='hour')
model.fit(x, num_loader_workers=2, verbose=True)
```
version:
python==3.7.11
torch==1.10.0
latest dev-all.txt requirements

Reply:
Mmmh no I don't get it. It might be an issue with OS/Python/PyTorch (not sure)
https://stackoverflow.com/questions/64772335/pytorch-w-parallelnative-cpp206
In any case, it won't show by default as `num_loader_workers=0` by default, so we can leave it and hope it'll solve itself in the future...